documentation/docs/getting_started/rust.mdx

```diff
@@ -8,79 +8,105 @@ keywords:
+
 # Getting Started with Rust
 
+The wallet.rs library is published on [crates.io](https://crates.io/crates/iota-wallet/versions). You can also download the source code from the [official GitHub repository](https://github.com/iotaledger/wallet.rs).
+
 ## Prerequisites
 
- _Rust_ and _Cargo_ are required to use wallet.rs. You can find installation instructions in the [Rust documentation](https://doc.rust-lang.org/cargo/getting-started/installation.html).
+You will need to install _Rust_ and _Cargo_ to use wallet.rs. You can find installation instructions in the [Rust documentation](https://doc.rust-lang.org/cargo/getting-started/installation.html).
 
 We recommend you update _Rust_ to the [latest stable version](https://github.com/rust-lang/rustup.rs#keeping-rust-up-to-date). The nightly version should be fine, but there is a chance some changes are not compatible.
 
 [_no_std_](https://docs.rust-embedded.org/book/intro/no-std.html) is not currently supported, but we are working on it, and we will provide it as a feature once the new implementation is ready.
 
 ### Dependencies
 
-[_cmake_](https://cmake.org/documentation/) and [_openssl_](https://www.openssl.org/docs/) are required to run Rust. To run the build process successfully using Cargo, you may need install additional build tools onto your system. 
+[_cmake_](https://cmake.org/documentation/) and [_openssl_](https://www.openssl.org/docs/) are required to run Rust. To run the build process successfully using Cargo, you may need install additional build tools onto your system.
 
-### Windows
+
+<Tabs>
+<TabItem value={'windows'} label={'Windows'}>
 
 You can download _cmake_ from the [official cmake website](https://cmake.org/download/).
-You can install _openssl_ with [vcpkg](https://github.com/microsoft/vcpkg) or [chocolatey](https://chocolatey.org/).
+You can install _openssl_ with [vcpkg](https://github.com/microsoft/vcpkg) or
+[chocolatey](https://chocolatey.org/).
 
-- Installing _openssl_ with _vcpkg_ :
+<Tabs>
+<TabItem value={'vcpkg'} label={'vcpkg'}>
 
-    ```
-    ./vcpkg.exe install openssl:x64-windows
-    ./vcpkg.exe integrate install
-    # you may want to add this to the system environment variables since you'll need it to compile the crate
-    set VCPKGRS_DYNAMIC=1
-    ```
+```bash
+./vcpkg.exe install openssl:x64-windows
+./vcpkg.exe integrate install
+# you may want to add this to the system environment variables since you'll need it to compile the crate
+set VCPKGRS_DYNAMIC=1
+```
 
-- Installing _openssl_ with _chocolatey_ :
+</TabItem>
+<TabItem value={'chocolatey'} label={'chocolatey'}>
 
-    ```
-    choco install openssl
-    # you may need to set the OPENSSL_ROOT_DIR environment variable
-    set OPENSSL_ROOT_DIR="C:\Program Files\OpenSSL-Win64"
-    ```
+```bash
+choco install openssl
+# you may need to set the OPENSSL_ROOT_DIR environment variable
+set OPENSSL_ROOT_DIR="C:\Program Files\OpenSSL-Win64"
+```
 
-### macOS
+</TabItem>
```

Comment:
Unrelated to this PR but can we make chocolatey the default and first entry? I think the chocolatey user base is substantially bigger than the vcpkg one 😅